Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -0,0 +1,1628 @@
+
+[* <COPYRIGHT>
+**   Copyright 2023, Jack Henry & Associates
+**
+** <NAME>
+**   RB.PREFACCESS.SSNMATCHING
```

Comment:
I like the RB in front. Will this PowerOn also be submitted to be in the SMRG PowerOn Library?

Reply:
Not sure with that but I guess it would make sense given that it's not restricted to just Banno use.